### PR TITLE
Saas 17 stock set warning if prod qty is not enough hch

### PR DESCRIPTION
**Pull request by hch-odoo**
Description of the issue/feature this PR addresses:
Task - https://www.odoo.com/web?#id=36014&view_type=form&model=project.task&action=327&menu_id=4720
Pad - https://pad.odoo.com/p/r.990e79d907a52382c9836a953008268e

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/stock/wizard/__init__.py [hunk 1]
```diff
@@ -6,4 +6,6 @@
 from . import make_procurement
 from . import procurement_orderpoint_compute
 from . import stock_immediate_transfer
-from . import stock_backorder_confirmation
\ No newline at end of file
+from . import stock_backorder_confirmation
+from . import stock_backorder_confirmation
```
Comment (mreficent) [inline addons/stock/wizard/__init__.py]: duplicated

File: addons/mrp/models/mrp_unbuild.py [hunk 2]
```diff
@@ -174,3 +174,17 @@ def _generate_move_from_bom_line(self, bom_line, quantity):
...
+        if avail_qty > self.product_qty:
+            self.action_unbuild()
+        else:
+            action = self.env.ref('stock.action_stock_insufficient_inventory').read()[0]
+            action.update({
+                'context': {
+                    'lot_id': self.lot_id.id,
+                    'default_unbuild_id': self.id}
+            })
+            return action
```
Comment (nhomar) [inline addons/mrp/models/mrp_unbuild.py]: thanks GOD

**Comment by nhomar** [comment]
Simple but extraordinarly necesary since time ago.

**Comment by jorenvo** [comment]
Will be done in saas-18 instead with https://github.com/odoo/odoo/pull/19655.

**Pull request closed without merging**